src/gazebo_motor_model.cpp

```diff
@@ -194,7 +198,13 @@ void GazeboMotorModel::UpdateForcesAndMoments() {
     gzerr << "Aliasing on motor [" << motor_number_ << "] might occur. Consider making smaller simulation time steps or raising the rotor_velocity_slowdown_sim_ param.\n";
   }
   double real_motor_velocity = motor_rot_vel_ * rotor_velocity_slowdown_sim_;
-  double force = real_motor_velocity * real_motor_velocity * motor_constant_;
+  double force = real_motor_velocity * std::abs(real_motor_velocity) * motor_constant_;
+  if(turning_direction_type_ == turning_direction::CCW || turning_direction_type_ == turning_direction::CW) {
+    // Not allowed to have negative thrust.
+    force = std::abs(force);
+  }
+
+  int turning_dir = turning_direction_type_ / std::abs(turning_direction_type_);
```

Comment:
Could we maybe not mix `turning_direction_type_` and the `turning_direction`? maybe add another boolean sdf element for "reversible"? I think this matches the mixer definitions better and makes it more explicit.